migrations/0023-add-feedback-table.ts

```diff
@@ -0,0 +1,67 @@
+import { MigrationInterface, QueryRunner, Table } from 'typeorm';
+
+export class SubmitFeedbackTable1603825929793 implements MigrationInterface {
+  public async up(queryRunner: QueryRunner): Promise<void> {
+    await queryRunner.createTable(new Table({
+      name: 'Feedback',
+      columns: [
+        {
+          name: 'id',
+          type: 'integer',
+          isGenerated: true,
+        },
+        {
+          name: 'uuid',
+          type: 'uuid',
+          isPrimary: true,
+          isGenerated: true,
+          generationStrategy: 'uuid',
+        },
+        {
+          name: 'user',
+          type: 'uuid',
+        },
+        {
+          name: 'title',
+          type: 'varchar(255)',
+        },
+        {
+          name: 'description',
+          type: 'text',
+        },
+        {
+          name: 'timestamp',
+          type: 'timestamptz',
+          default: 'CURRENT_TIMESTAMP(6)',
+        },
+        {
+          name: 'acknowledged',
```

Comment:
I don't think we're doing a "mark as spam" thing, since Kendall mentioned it would be dealt on a case-by-case basis and since we're only rewarding points on acknowledgement rather than submission. Do you think we should have it still?